src/dstack/_internal/cli/utils/fleet.py

```diff
@@ -51,6 +53,7 @@ def get_fleets_table(fleets: List[Fleet], verbose: bool = False) -> Table:
 
             row = [
                 fleet.name if i == 0 else "",
+                fleet.spec.configuration.reservation if i == 0 else "",
```

Comment:
Only include reservation value if verbose. Otherwise it shifts all the values without verbose:

```
✗ dstack fleet
 FLEET                  INSTANCE  BACKEND  RESOUR…  PRICE     STATUS  CREATED      
 popular-rattlesnake-1            0        aws      2xCPU,    $0.107  provis…  now 
                                           (eu-we…  8GB,                           
                                                    100.0GB                        
                                                    (disk) 
```

Reply:
Fixed in [5647aff](https://github.com/dstackai/dstack/pull/1977/commits/5647aff2094cde0300eb360ffecfcf47bee3335f)